portal/src/texts/designer/gettingStarted.md

```diff
@@ -1,19 +1,25 @@
 ## Kom i gang
 
-Er du utvikler eller designer og ivrig etter å komme i gang, finner du de ressursene du trenger her. Vi ønsker at det skal være så enkelt så mulig for dere å ta i bruk Jøkul.
+Er du utvikler eller designer og ivrig etter å komme i gang? Her finner du de ressursene du trenger! Vi ønsker at det skal være så enkelt som mulig for alle å ta i bruk Jøkul.
 
 ### Om Jøkul
 
-Jøkul er designsystemtet for Fremtind. Det utvikles stadig og målet med systemet er at det skal gjøre det lettere for alle i Fremtid å utvikle og vedlikeholde løsningene sine.
+Jøkul er designsystemet til Fremtind. Det er under stadig utvikling,og målet med systemet er at det skal gjøre det lettere for alle fagteam i Fremtind å utvikle og vedlikeholde løsningene sine.
+
 For at Jøkul skal kunne oppfylle det målet, er det viktig at teamene som bruker Jøkul bidrar tilbake.
 
 Vi har utviklet og samlet komponenter i GitHub og i et eget designbibliotek i Figma.
 
 ### For designere
 
 Vi lager skisser og prototyper i Figma.
-Dette trenger du
-Det er enkelt å komme i gang med design for Fremtind. Sett deg inn i fargepaletten og prinsippene våre. Studer gjerne eksemplene her i Jøkul før du starter, så får du et overblikk over hvordan Fremtinds design er tenkt og hvordan vi implementerer. Vi lager komponenter både til interne fagsystemer og kundeløsninger, så løsningene er varierte.
-Du trenger tilgang til:
-Designbiblioteket vårt. Send en e-post til mikkel.blytt@fremtind.no.
-Skrifttypen vår, Akkurat Pro. Last ned fonten her
+
+#### Dette bør du gjøre først
+- Sett deg inn i fargepaletten vår.
+- Studer prinsippene våre og se på eksempler på komponenter, typografi, luft, bildebruk osv. Da får du et overblikk over hvordan designet er tenkt. Vi lager komponenter både til interne fagsystemer og kundeløsninger, så løsningene er varierte.
+
+#### Du trenger tilgang til
+- designbiblioteket vårt 
+- skrifttypen vår, Fremtind grotesk 
+
+Send en e-post til mikkel.blytt@fremtind.no, så ordner han det på 1-2-3!
```

Comment:
Tror ikke det er så lurt å publisere mailen til mikkel. Her burde vi kunne funnet et abstraksjonslag. Spørs om noen husker å oppdatere denne den dagen Mikkel blir designsjef for Statoil eller noe sånt.

Reply:
Da må dere opprette en aliasadresse: hva med designbibliotek@fremtind.no?